openverse_catalog/dags/common/loader/provider_details.py

```diff
@@ -24,7 +24,7 @@
 NYPL_DEFAULT_PROVIDER = "nypl"
 RAWPIXEL_DEFAULT_PROVIDER = "rawpixel"
 SCIENCE_DEFAULT_PROVIDER = "sciencemuseum"
-STATENS_DEFAULT_PROVIDER = "statensmuseum"
+SMK_DEFAULT_PROVIDER = "statensmuseum"
```

Comment:
Nit, statensmuseum basically just means 'state museum' and doesn't really represent this source well. Is there any consequence to changing this value?

Reply:
This value reflects the `provider` field in our actual data I believe. If we wanted to update this, we'd need to update all associated data (and anything that references this provider in the API and the frontend).

Reply:
I'm okay with that, as I don't believe this value is hardcoded anywhere, so we'd just need to update it in the Django Admin UI, which we need to do anyway to enable it.

Reply:
```bash
~/Code/openverse
❯ rg 'statensmuseum'
catalog/openverse_catalog/dags/common/loader/provider_details.py
27:SMK_DEFAULT_PROVIDER = "statensmuseum"
```

quick safety check (this is searching all our repos)

Reply:
Yeah, I kept this as is because it's the field in our actual data. If we update this and do nothing, for example, we'd get a bunch of `duplicate key` errors as soon as we try to ingest data. I assumed it would also require changes on the API/frontend, but since it seems it doesn't, I think all we'd need to do is:
* `UPDATE image SET provider='smk' WHERE provider='statensmuseum'` in production
* Update to SMK & enable in the Django admin UI

Not so bad :) I'll update it.

Reply:
Alternatively we can just [delete everything for statensmuseum](https://github.com/WordPress/openverse-catalog/pull/506#issuecomment-1129122059) and let the next data pull re-populate it with the correct provider name and urls.